### ISSUE TITLE
DaCy large requires "protobuf>=3.17.3,<3.18.0" to run

### ISSUE THREAD

**Issue by KennethEnevoldsen** [centre-for-humanities-computing/DaCy#218]
## How to reproduce the behaviour

```bash
$ pip install dacy
$ python
>>> import dacy
>>> nlp = dacy.load("large")
[...]
ImportError: 
XLMRobertaConverter requires the protobuf library but it was not found in your environment. Checkout the instructions on the
installation page of its repo: https://github.com/protocolbuffers/protobuf/tree/master/python#installation and follow the ones
that match your environment. Please note that you may need to restart your runtime after installation.
```

This can be fixed using:
```bash
$ pip install dacy
$ pip install "protobuf>=3.17.3,<3.18.0"
$ python
>>> import dacy
>>> nlp = dacy.load("large")
```

## Your Environment

- **DaCy Version Used:** 2.4.0
- **spaCy version:** 3.5.1
- **Platform:** macOS-13.0.1-arm64-arm-64bit
- **Python version:** 3.10.10
- **Pipelines:** da_dacy_large_trf (0.1.0), da_dacy_small_trf (0.1.0)

**Comment by KennethEnevoldsen** [comment]
It is intended that this bug is fixed during the next iterations of models, where the large model will likely be replace by the large Danish DFM model instead of the multilingual model.

**Comment by KennethEnevoldsen** [comment]
solved in the new 0.2.0 models
